backend/src/main/java/com/woowacourse/gongseek/comment/domain/repository/CommentRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.woowacourse.gongseek.comment.domain.repository;
+
+import com.woowacourse.gongseek.comment.domain.Comment;
+import java.util.List;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface CommentRepository extends JpaRepository<Comment, Long> {
+
+    List<Comment> findCommentsByArticleId(Long articleId);
```

Comment:
Comment repo니깐 찾아도 Comment일것 같아서 
```suggestion
    List<Comment> findAllByArticleId(Long articleId);
```
로 하는건 어떤가요?🙄

Reply:
넹 수정했습니당~